### PR TITLE
Add widget CTA and instructions polish

### PR DESCRIPTION
**Pull request by subsymbolic**
Task/Issue URL: 
- CTA split screen layout: https://app.asana.com/0/361428290920652/1109550783732474
- Landscape widget instructions: https://app.asana.com/0/361428290920652/985536855665868
- Divider color: https://app.asana.com/0/361428290920652/997377039641955

**Description**:
Additional polish to the "add widget" instructions and CTA screens including
- Suppress landscape of CTA when in portrait split screen view
- Add landscape screen for widget instructions
- Update color of instructions divider

**Steps to test this PR**:

**TEST 1: Split screen CTA**
1. Install a fresh version of the app
1. Complete onboarding
1. Note the CTA appears
1. Play with the app in split screen and ensure the CTA doesn't switch to landscape when there isn't enough room, see https://app.asana.com/0/361428290920652/1109550783732474 for an example image of what can happen in develop

**TEST 2: Instructions screen landscape and divider color**
1. Press the "Show Me" button on the widget CTA
1. Put your device in landscape and note that the layout is updated
1. Note that the divider above the buttons is now a black transparent color rather than a solid grey, see https://app.asana.com/0/361428290920652/997377039641955 for more info

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

**Comment by CDRussell** [comment]
screenshot_1550143446

_Nexus S Emulator_

It seems odd that the buttons can be below the fold. I feel like they should be anchored to the bottom, and the rest of the layout fits in the remaining space, rather than the buttons being pushed off-screen.

This is a pretty minor complaint though - I doubt too many devices are going to see this.

File: app/src/main/res/layout/activity_add_widget_instructions.xml [hunk 1]
```diff
@@ -20,177 +22,36 @@
...
-                android:id="@+id/homeButton"
-                style="?android:attr/borderlessButtonStyle"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:text="@string/addWidgetInstructionsButtonGoHome"
-                android:textColor="@color/white"
-                android:textSize="13sp"
-                app:layout_constraintEnd_toEndOf="parent"
-                app:layout_constraintTop_toBottomOf="@id/instruction3" />
+            android:layout_centerInParent="true"
```
Comment (CDRussell) [inline app/src/main/res/layout/activity_add_widget_instructions.xml]: This is flagged as invalid layout param

Comment (subsymbolic) [reply app/src/main/res/layout/activity_add_widget_instructions.xml]: Ahh thanks, that's not needed! I've removed it.

**Comment by subsymbolic** [comment]
Many thanks for your review @CDRussell. I've added your landscape feedback for small devices into LHF polish.

**Pull request merged**